src/Rule/EnsureExplicitNullableTypes.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of DOCtor-RST.
+ *
+ * (c) Oskar Stark <oskarstark@googlemail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Rule;
+
+use App\Attribute\Rule\Description;
+use App\Value\Lines;
+use App\Value\NullViolation;
+use App\Value\RuleGroup;
+use App\Value\Violation;
+use App\Value\ViolationInterface;
+
+#[Description('Ensure explicit nullable types in method arguments.')]
+class EnsureExplicitNullableTypes extends AbstractRule implements LineContentRule
+{
+    public static function getGroups(): array
+    {
+        return [
+            RuleGroup::Symfony(),
+        ];
+    }
+
+    public function check(Lines $lines, int $number, string $filename): ViolationInterface
+    {
+        $lines->seek($number);
+        $line = $lines->current();
+
+        if (!str_contains((string) $line->clean(), ' = null')) {
+            return NullViolation::create();
+        }
+
+        $pattern = '/([?]?\w+)\s+\$(\w+)\s*=\s*null(?=\s*[,\)])/';
+
+        if ($matches = $line->clean()->match($pattern)) {
+            $types = $matches[1];
+
+            // ?int $id = null
+            if (str_starts_with($types, '?')) {
+                return NullViolation::create();
+            }
+
+            // int|null $id = null
```

Comment:
Do we accept this syntax in symfony-docs ?

If not, we could remove it from rule, or make rule configurable (see `src/Rule/DeprecatedDirectiveMajorVersion.php` wichi implements `Configurable`)

@OskarStark What's your opinion?

Reply:
In Symfony we use ?int but also int|string|null 

Let's keep it for now

Reply:
Probably not so frequent for just one type, but it's more to illustrate the syntax when multiple types + null are set. 

```php
final class MonotonicClock implements ClockInterface
{
    // ...

    public function __construct(\DateTimeZone|string|null $timezone = null)
    {
```

I'm replacing the comment with "int|string|null" to be more accurate